### DISCUSSION TITLE
Game, information window position, should we adjust the position of the info window as in the original game

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#5122] [Ideas]
In the original game, in most cases, the top of the information window is at the same position as the window from which it is opened.
Should we adjust the position of the info window as in the original game for windows smaller in height than the window from which it is opened?
Windows that are larger in height should remain centered as they are now.
I find that the current way of centering all information windows is not always visually appealing.

In original game:

https://user-images.githubusercontent.com/43583994/158058123-680fe15f-96b5-44f7-b39c-2092fa85684c.mp4

https://user-images.githubusercontent.com/43583994/158058130-59f84440-bbcc-400f-a6d3-635fb287e99b.mp4

In fheroes2:

https://user-images.githubusercontent.com/43583994/158058155-03ed7e77-812e-4cfd-9c95-27e5c6b88be9.mp4

Version 0.9.12 5071

**Comment by Branikolog** [comment]
Hi, @LeHerosInconnu !
I better like an idea of placing all windows using the same rules: in the center of the screen. Also, (I don't know if this could actually happen) in a case we have basic window rather small, being shown in the center of the screen, and the pop up info window being too large, will lead a pop up window to be shifted too much to the bottom (or even out of the screen) if the description takes much place inside it.

**Comment by LeHerosInconnu** [comment]
It's okay the way it works currently.
And @Branikolog has a good point. :)
I close the discussion.
